lifecycle_policy.md

```diff
@@ -0,0 +1,168 @@
+# High Performance Software Foundation Project Lifecycle Policies
+
+## I. Overview
+
+*This governance policy describes how an open source project can formally join the High Performance Software Foundation via the Project Proposal Process. It describes the Stages a project may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a project to move from one stage to another. It also describes the Periodic Review Process (PRP) to reassess project stage assignments.*
+
+Project progression - movement from one stage to another - allows projects to participate at the level that is most appropriate for them given where they are in their lifecycle. Regardless of stage, all High Performance Software Foundation projects benefit from a deepened alignment with existing projects, and access to mentorship, support, and foundation resources.
+
+Capitalized terms not otherwise defined in this Project Lifecycle Policy have the meanings ascribed to them in the Charter of the High Performance Software Foundation.
+
+
+## II. Project Proposal Process
+
+### Introduction
+This governance policy sets forth the proposal process for projects to be accepted into the High Performance Software Foundation. The process is the same for both existing projects which seek to move into the High Performance Software Foundation and new projects to be formed within the High Performance Software Foundation.
+
+### Project Proposal Requirements
+Projects must be formally proposed via GitHub. Project proposals submitted to the High Performance Software Foundation should provide the following information to the best of their ability:
```

Comment:
Another change I would propose that would require TAC approval is to mention that the repository has an issue template for projects to construct their proposals. Something like

> Projects must be formally proposed via GitHub. The TAC Github repository includes an issue template to help projects format their proposals. Project proposals submitted to the High Performance Software Foundation